packages/ui/forms/create-account/index.tsx

```diff
@@ -35,8 +35,10 @@ const loadingContext = new LoadingContext()
 
 interface Props {
   onNickTextInputBlur: (nickname: string) => void
+  onNickTextChange: (email: string) => void
```

Comment:
to make this a bit clearer, change (email: string) to (nickname: string)

Reply:
good point, oops